ingestion-beam/src/main/java/com/mozilla/telemetry/Decoder.java

```diff
@@ -51,7 +51,7 @@ public static PipelineResult run(String[] args) {
    */
   public static PipelineResult run(DecoderOptions.Parsed options) {
     final Pipeline pipeline = Pipeline.create(options);
-    final List<PCollection<PubsubMessage>> errorCollections = new ArrayList<>();
+    final List<PCollection<PubsubMessage>> failureCollections = new ArrayList<>();
```

Comment:
I thought calling this `failureCollections` is more correct since it is now handling failures